index.js

```diff
@@ -75,6 +76,14 @@ const factory = options => {
     opts.issWhitelist.indexOf(token.payload.iss) > -1 ||
     Promise.reject(new IssWhitelistError(`iss '${token.payload.iss}' not in issWhitelist`))
 
+  const checkExp = ({ ignoreExpiration }) => ({ payload: { exp } }) => {
+    if (ignoreExpiration) return
+
+    if (exp && exp > (Date.now() / 1000)) {
```

Comment:
@jrpz Because this is backwards. Note the casing of `jwt expired` in the assertion doesn't match the casing :point_down: , because its coming from `jsonwebtoken.verify`.

```suggestion
    if (exp && exp <= (Date.now() / 1000)) {
```

Reply:
Are you talking about this bit: https://github.com/articulate/authentic/pull/23/files#diff-5bb8db779819ddef5956a5d9d5949c05ef7445237656ca37bf2f02720271440bR179

Reply:
I am... 😕 I wonder why that's passing? Maybe it has to do with how `verify` composed & ramda-ified.